circuits/circom/prover/.gitattributes

```diff
@@ -0,0 +1 @@
+*.zkey filter=lfs diff=lfs merge=lfs -text
```

Comment:
didn't know `.gitattributes` even existed!

Reply:
Yeah I just learned about it too! But I should remove it anyway, because the prover key ended up being too big even for git large file storage (lfs).